packages/syft/src/syft/service/request/request.py

```diff
@@ -1143,7 +1162,7 @@ class UserCodeStatusChange(Change):
 
     @property
     def code(self) -> UserCode:
-        return self.linked_user_code.resolve
+        return self.linked_obj.resolve
```

Comment:
That's my bad, will revert it to linked_user_code.
And yes its possible to rename, we have a `rename` transform. So in the new versionUserCodeStatusChange, linked_obj is no longer used and being renamed to `linked_user_code ` ?

Reply:
Both are used, we decoupled code from status, and now mutate the UserCodeStatusCollection instead of the UserCode in `_run`. 

Probably can refactor it later to not link to the UserCode and only to the UserCodeStatusCollection, but decided to keep both in for now